plugin/documents.py

```diff
@@ -144,7 +144,7 @@ class DocumentSyncListener(sublime_plugin.ViewEventListener, AbstractViewListene
     ACTIVE_DIAGNOSTIC = "lsp_active_diagnostic"
     code_actions_debounce_time = 800
     color_boxes_debounce_time = 500
-    highlights_debounce_time = 300
+    highlights_debounce_time = 500
```

Comment:
I'm wondering if we should take the opportunity to clean it up and just use one timeout for all (and potentially make it customizable).

I do think that VSCode feels snappier in showing diagnostics and code actions, probably because it has a lower timeout so I would want to try with lower values also. But that can be considered a separate request. For now maybe just use the same timeout for both those two and also the text change timeout so that those stay in sync?